src/hooks/debilidades.ts

```diff
@@ -58,27 +19,31 @@ export const usePokemonWeaknesses = (pokemon: Pokemon | undefined) => {
   return weaknesses;
 };
 
-const PokemonDetails = async ({ pokemon }: { pokemon: Pokemon }) => {
```

Comment:
Minor thing: in JavaScript, capitalized function names usually carry special meaning. 

It's a **convention** where functions intended to be classes are named with a capital letter. 

If I read a function named `PokemonDetails()`, I'd expect one of the following:

- its a component meant to be renderer as `<PokemonDetails />`
- its a function meant to be used as a class like: `new PokemonDetails()`
- its a function and the original writer is telling me that is creating some kind of object and probably has something like: `PokemonDetails.create()`

***

TL;DR if it's a regular function name it lowercase xD